greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md

```diff
@@ -7,4 +7,5 @@
 πως είναι ώρα να βγεις από εκεί. Πόρτα δεν υπάρχει οπότε πρέπει
 να χρησιμοποιήσεις κάτι από αυτά που έχεις. Τι κάνεις;
 
-[Ανάβεις τον αναπτήρα](set_it_on_fire/set_it_on_fire.md)
\ No newline at end of file
+[Ανάβεις τον αναπτήρα](set_it_on_fire/set_it_on_fire.md)
+[Πυρώνεις το πιρούνι με τον αναπτήρα](fork_it/fork_it.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.